tests/vds/documentapi/document_api_v1_part1.rb

```diff
@@ -10,10 +10,10 @@ def test_get_with_number
   end
 
   def test_visit_with_number
-    response = api_http_get('/document/v1/storage_test/music/number/')
+    response = api_http_get('/document/v1/storage_test/music/docid/')
```

Comment:
Does test name `test_visit_with_number` make sense anymore? Seems more like  `test_continuation_token_returned_after_visiting_single_document`

Reply:
Well, the document is stored with a number in its ID, so I thought leaving it like this was OK. You're the expert here, though :)

Reply:
It's not testing `/number/` anymore (and arguably never really did), so in that case it should be renamed `test_visit_of_document_with_number_location_in_id` or similar to avoid ambiguities. If we have another test that explicitly tests using `/number/` I'd still suggest renaming to the first suggestion.

Reply:
Perhaps this test was meant to include the number ...

Reply:
Ah, well it does that a few lines further down. I guess _that_'s reason to leave the name in place :)